### ISSUE TITLE
Mobile Emulator 10.0.14393.0 WVGA 4 inch 1GB (Windows AU SDK)

### ISSUE THREAD

**Issue by dg2k** [Windows-XAML/Template10#1178]
I notice something strange with a 4 inch Mobile Emulator (that comes with Windows AU SDK)  as shown in the image below (PageHeader control moves down). I  do not think the problem has to do with T10, rather with the Emulator. I coudn't confirm this for sure as I do not have a 4 inch device. All the other 10.0.14393.0 emulators display correctly.

Can someone with 4 inch device (e.g., **Lumia 430**, **Lumia 435**) please confirm if **Hamburger Project** displays okay when running under VS 2015 Update 3, with project **Target version** as **Build 14393**?

![t10](https://cloud.githubusercontent.com/assets/14026786/17648816/6acf85b2-621b-11e6-8f23-559aff3387e3.PNG)

**Comment by RagBuster** [comment]
I have a project which isn't using Template 10 but I am experiencing visual problems like you describe at the top and also at the bottom, looks to only effect the 4" emulator. Looks like the page content has been shifted down by 1cm where content goes below the bottom appbar.

**Comment by dg2k** [comment]
Thanks for the feedback. This is in line with my suspicion that it's the 4" emulator that may be buggy. Yes, I also see the visual problem at the bottom.

**Comment by dubstylee** [comment]
I noticed a similar thing with the emulators, but the issue is not there on a device. That white gap at the top is simply where the carrier name/wifi signal/etc. would be normally.

**Comment by FahadMustafa** [comment]
I am seeing this issue on actual devices (Lumia 950 and Lumia 550) running T10 v1.1.11. Both devices are running Windows AU.

**Comment by dg2k** [comment]
I tested on Lumia 950 XL and all is fine (**later update:** BUT the VisualState problem is there!). Not sure if resolution difference between 950 and 950 XL makes any difference.

Based on my initial test with Build 14393, I was of the opinion that it was only the 4 inch emulator that was buggy but I could be wrong.

**UPDATE**
Further to what I reported in the above, I note further issues with 14393 for Mobile. The Visual States that work with 10586 do not work with 14393 as they should. Fortunately, the test scenarios are as straightforward as switching between 10586 and 14393 emulators for the same code, and differences can be observed clearly with 14393 not behaving as expected. This is strange.
